### PR TITLE
More Anime Choices

### PR DESCRIPTION
**Pull request by lmenvs**
# About The Pull Request
Adds Fox, Wolf, and Shark anime personas.  Ported from TG which was ported from Skyrat which was ported from who-the-fuck-knows.

Additionally adds more functionality to the trait; the items get their own type of coloration (you can still hit cancel on the color picker menu to have it default to your hair color)

## Why It's Good For The Game
More weeb choices, makes the trait less like a "but this is just felinids" and more like an actual trait.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
image

</details>

## Changelog

:cl:
add: More anime trait options
add: Anime sprite accessories have their own separate color option!
fix: Felinid Weebs can eat junkfood again
/:cl:

### CODE REVIEW

**Comment by unit0016** [comment]
I was summoned to this PR for unknown reasons.

File: monkestation/code/modules/surgery/organs/tails.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/obj/item/organ/tail/monkey
+	name = "monkey tail"
+	desc = "A severed monkey tail. If found, please call 1-800-OOK-EEEK"
+	icon = 'monkestation/icons/mob/mutant_bodyparts.dmi'
+	icon_state = "monkey"
+	color = "#5f4637" //for debugging
```
Comment (nednaZ) [inline monkestation/code/modules/surgery/organs/tails.dm:6]: Debugging still needed?

**Review by nednaZ — COMMENTED** [review]
Will review in more depth later, just a quick skim.

File: monkestation/code/modules/surgery/organs/tails.dm (hunk 1 above)
Comment (lmenvs) [reply monkestation/code/modules/surgery/organs/tails.dm:6]: Ehh this just colors the dropped tail icon, can probably stay there. can remove the comment tho

**Review by nednaZ — APPROVED** [review]
Oh my god I hate it so much.

**Pull request merged**